src/layer/S3SLSTRLayer.ts

```diff
@@ -32,13 +32,14 @@ export class S3SLSTRLayer extends AbstractSentinelHubV3Layer {
   ) {
     super(instanceId, layerId, evalscript, evalscriptUrl, dataProduct, title, description);
     this.maxCloudCoverPercent = maxCloudCoverPercent;
-    // images that are not DESCENDING are blank, so we can hardcode this:
+    // images that are not DESCENDING appear empty:
     this.orbitDirection = OrbitDirection.DESCENDING;
     this.view = view;
   }
 
   protected async updateProcessingGetMapPayload(payload: ProcessingPayload): Promise<ProcessingPayload> {
     payload.input.data[0].dataFilter.maxCloudCoverage = this.maxCloudCoverPercent;
+    payload.input.data[0].dataFilter.orbitDirection = this.orbitDirection;
```

Comment:
Not really a comment regarding the code directly, but I would assume `view` would have to be added to payload at some point as well? Right now we search by `NADIR`, which is also the default for Processing API.